.eslintrc.json

```diff
@@ -10,5 +10,9 @@
   "env": {
     "node": true,
     "es6": true
+  },
+
+  "rules": {
+    "no-unused-vars": [2, {"vars": "all", "args": "none"}]
   }
-}
\ No newline at end of file
+}
```

Comment:
We could also add more rules, if we want to switch to `let`+`const` instead of `var`:

```json
  "rules": {
    "indent": [2, 2],
    "no-var": 1,
    "prefer-const": 1
  }
```